### PR TITLE
Add OTP 18

### PR DESCRIPTION
**Pull request by surik**
Relate to #33

### CODE REVIEW

File: src/eradius_counter.erl [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 %% @doc initialize a counter structure
 init_counter(Key = {_ServerIP, ServerPort}) when is_integer(ServerPort) ->
-	#server_counter{key = Key, upTime = now(), resetTime = now()};
+	#server_counter{key = Key, upTime = os:timestamp(), resetTime = os:timestamp()};
```
Comment (RoadRunnr) [inline src/eradius_counter.erl]: os:timestamp/0 is not the same a erlang:now/0 was, simply replacing one with other might not be the correct solution.

See http://www.erlang.org/doc/apps/erts/time_correction.html#id76661 and http://www.erlang.org/doc/apps/erts/time_compat.erl for better ways to handle this.

**Comment by surik** [comment]
@RoadRunnr thanks for your remarks, I've fixed it.

**Comment by RoadRunnr** [comment]
@surik: almost there, we still need to support tetrapak and it's check target. It now failes on R17 with:

```
$ tetrapak check
== check:xref ====================
Undefined Functions called:
  erlang:timestamp/0
    by eradius_lib:timestamp/0
Error: xref error
```

Please add xref override to tetrapak/config.ini, something like:

```
[xref]
ignore_undef = [ { erlang, timestamp, 0 } ]
```

**Comment by surik** [comment]
done

**Pull request merged**